www/js/survey/enketo-survey-service.js

```diff
@@ -103,22 +103,36 @@ angular.module('emission.enketo-survey.service', [
     return answer;
   }
 
+  function _processSurveyAnswers(answers, key, opt_populateLabels = false) {
```

Comment:
why do you need this new function? It seems like it is just code restructuring...

Reply:
It is shared between `getAllSurveyAnswers` and `getAllLocalSurveyAnswers`.

Reply:
Yeah I figured that out. But it would have been helpful to state that upfront instead of making me try and understand what you were doing...